src/components/Oracles/MarchMadness/BracketsView.tsx

```diff
@@ -112,40 +113,75 @@ const SeedTeamWrapper = ({
 }) => {
   const isWinner = winner === team?.alias
   return (
-    <SeedTeam style={{ color: isWinner ? 'green' : 'white' }}>
-      <div style={{ fontWeight: isWinner ? '700' : '400' }}>
-        {isWinner ? '🏆' : ''}
+    <SeedTeam className={isWinner ? 'winner' : 'no-winner'}>
+      <Flex alignItems="center" fontWeight={500} gap="1" fontSize="xs">
         {team?.alias || 'NO TEAM '} {team?.seed && `(${team?.seed})`}
-      </div>
+        {isWinner ? <Icon as={RiTrophyFill} fill="yellow.500" /> : ''}
+      </Flex>
 
       <div style={{ display: 'inline' }}>{team_points}</div>
     </SeedTeam>
   )
 }
 
-const CustomSeed = ({ seed, breakpoint }: IRenderSeedProps) => (
-  <Seed mobileBreakpoint={breakpoint} style={{ fontSize: 14 }}>
-    <SeedItem>
-      <div>
-        {seed ? (
-          <>
-            <SeedTeamWrapper
-              winner={seed.winner}
-              team={seed.teams[0] as Team}
-              team_points={seed.game.home_points}
-            />
-            <SeedTeamWrapper
-              winner={seed.winner}
-              team={seed.teams[1] as Team}
-              team_points={seed.game.away_points}
-            />
-          </>
-        ) : null}
-      </div>
-    </SeedItem>
-    <div className="sc-gKsewC ha-DNGW">{seed.date}</div>
-  </Seed>
-)
+const CustomSeed = ({ seed, breakpoint }: IRenderSeedProps) => {
+  const eventDateObj = new Date(seed.date as string)
+
+  const EventDate = new Intl.DateTimeFormat('en-US', {
+    weekday: 'short',
+    year: 'numeric',
+    month: 'short',
+    day: 'numeric',
+  }).format(eventDateObj)
+
+  const EventTime = eventDateObj.toLocaleTimeString('en-US', {
+    minute: '2-digit',
+    hour: '2-digit',
+  })
+
+  return (
+    <Seed mobileBreakpoint={breakpoint} style={{ fontSize: 14 }}>
+      <Box
+        border="1px solid"
+        borderColor="bracketsCardBorder"
+        borderRadius="6px"
+        w="full"
```

Comment:
The card is overflowing its borders, i believe overflow hidden over this one might fix.

<img width="711" alt="CleanShot 2022-12-21 at 17 25 12@2x" src="https://user-images.githubusercontent.com/52039218/208899630-0ba5bfe3-efd2-44ea-9b9f-7744d773ddde.png">

Reply:
Ha! Okay, I'll push the fix now.